dockerfiles/steps/step-epub.bash

```diff
@@ -6,7 +6,13 @@ set -Eeuxo pipefail
 # Ensure $IO_EPUB is empty
 [[ -d $IO_EPUB ]] && rm -rf ${IO_EPUB:?}/*
 
-node --unhandled-rejections=strict "$JS_UTILS_STUFF_ROOT/bin/bakery-helper" epub ./ "$IO_EPUB/"
+extra_vars=""
+(( JS_DEBUG==1 )) && {
```

Comment:
Similarly `# LCOV_EXCL_START` should fix this

Reply:
Also, rather than remembering to add this block everywhere node is called (like in mathify, the docx conversion, adding source information while assembling, ...), it [seems](https://stackoverflow.com/a/71688732) an environment variable can be set in a common place like the docker-entrypoint:

```
NODE_OPTIONS='--inspect-brk=0.0.0.0:9229'
```